### PR TITLE
181608667 remove destroy all from tests

### PR DESCRIPTION
**Pull request by maciejkocylapc**
#### What's this PR do?
- remove destroy_all and delete_all from tests
- remove fixtures

#### How should this be manually tested?
- run bundle install
- run tests

#### What are the relevant tickets?
- URL to the PivotalTracker ticket

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: test/logic/validator_score_v1_logic_test.rb [hunk 1]
```diff
@@ -8,9 +8,21 @@ class ValidatorScoreV1LogicTest < ActiveSupport::TestCase
 
   def setup
     # Create our initial payload with the input values
+    @batch = create(:batch, network: "testnet")
```
Comment (kamtarnowski) [inline test/logic/validator_score_v1_logic_test.rb:11]: No need network reference. Testnet is default.

File: test/logic/validator_score_v1_logic_test.rb [hunk 2]
```diff
@@ -62,28 +73,30 @@ def setup
                 .then(&validators_get)
                 .then(&block_vote_history_get)
 
+    puts p.code
```
Comment (kamtarnowski) [inline test/logic/validator_score_v1_logic_test.rb:76]: Let's remove of that.

File: test/logic/validator_score_v1_logic_test.rb [hunk 3]
```diff
@@ -299,7 +308,13 @@ def setup
   end
 
   test 'assign_block_history_score' do
-    create(:validator_block_history, network: 'testnet', batch_uuid: '1234')
+    create(
+      :validator_block_history,
+      network: 'testnet',
```
[the remark this answers is not in the document]
Comment (kamtarnowski) [reply test/logic/validator_score_v1_logic_test.rb:313]: Let's address printer issue.

**Review by kamtarnowski — CHANGES_REQUESTED** [review]
There is also occurrence of destroy_all in test/queries/validator_query_test.rb regarding Validator class. Please address it as well.

File: test/controllers/api/v1/validator_block_histories_controller_test.rb [hunk 4]
```diff
@@ -4,132 +4,135 @@
...
-    validator = create(:validator, :with_score, account: "TestAccount")
-    create(:vote_account, validator: validator)
-    create(:validator_block_history, validator: validator)
-
-    get "/api/v1/validator_block_history/testnet/#{validator.account}", headers: { "Token" => @user.api_token }
-
-    json_response = response_to_json(@response.body)
+module Api
+  module V1
+    class ValidatorsControllerTest < ActionDispatch::IntegrationTest
```
Comment (kbiala) [inline test/controllers/api/v1/validator_block_histories_controller_test.rb:9]: This one should be called ValidatorBlockHistoriesControllerTest

File: test/logic/validator_score_v1_logic_test.rb [hunk 5]
```diff
@@ -1,58 +1,68 @@
...
     p.payload[:validators].each do |validator|
       assert_not_nil validator.validator_score_v1
       assert_equal @initial_payload[:network], validator.validator_score_v1.network
     end
   end
 
-  test 'block_vote_history_get' do
-    # create enough records to confirm via logs we don't have an N+1 query when
+  test "block_vote_history_get" do
+    # create enough records to confirm via logs we don"t have an N+1 query when
```
Comment (kbiala) [inline test/logic/validator_score_v1_logic_test.rb:50]:
```suggestion
    # create enough records to confirm via logs we don't have an N+1 query when
```

File: test/test_helper.rb [hunk 6]
```diff
@@ -25,6 +26,25 @@
...
 
+DatabaseCleaner.strategy = :truncation
+
+module AroundEachTest
+  def before_setup
+    super
+    DatabaseCleaner.start
+  end
+
+  # def after_teardown
```
Comment (kbiala) [inline test/test_helper.rb:37]: You can remote commented part

**Review by kbiala — CHANGES_REQUESTED** [review]

**Pull request merged**